skimage/segmentation/tests/test_watershed.py

```diff
@@ -392,6 +392,18 @@ def test_watershed12(self):
         for lab, area in zip(range(4), [34,74,74,74]):
             self.assertTrue(np.sum(ws == lab) == area)
 
+    def test_watershed_input_not_modified(self):
+        """Test on markers
+
+        This is here to ensure input markers is not modified
+        """
```

Comment:
```suggestion
        """Test to ensure input markers are not modified."""
```

Reply:
Hmm, maybe my suggestion is not ideal, because it sounds like "Test" is a noun, not a verb... Otherwise, "Test that input markers are indeed not modified." :wink: